pyiron_base/project/generic.py

```diff
@@ -1458,3 +1460,34 @@ def _update_jobs_in_old_database_format(self, job_name):
             elif db_entry_in_old_format:
                 for entry in db_entry_in_old_format:
                     self.db.item_update({"project": self.project_path}, entry["id"])
+
+
+    def packing(self, destination_path, csv_file_name='export.csv', compress=True):
+        """
+        by this funtion, the job table is exported to a csv file
+        and the project directory is copied and compressed (by default) to a file.
+        
+        Args:
+        destination_path (str) gives the ralative path, in which the project folder is copied and the compressed
+        csv_file_name (str) is the name of the csv file used to store the project table.
+        compress (boolian), if true, the function will compress the destination_path to a tar.gz file 
+        """
+        export_archive.copy_files_to_archive(self.path, destination_path,compressed=compress)
+        df = export_archive.export_database(self,self.path,destination_path)
+        df.to_csv(csv_file_name)
+
+
+    def unpacking(self, origin_path, csv_file_name='export.csv', compress=True):
```

Comment:
I guess the `origin_path` should allow a relative path, an absolute path as well as a project object. So we should test that the variable is converted correspondingly.

Reply:
I think with the addition of these new line, the `origin_path` can be a relative path, absolute path, or a Project() object.
[https://github.com/pyiron/pyiron_base/blob/748f6e24949d973fea457c76159032901880727a/pyiron_base/archiving/import_archive.py#L36](https://github.com/pyiron/pyiron_base/blob/748f6e24949d973fea457c76159032901880727a/pyiron_base/archiving/import_archive.py#L36)